tests/output.rs

```diff
@@ -67,9 +66,35 @@ fn strip_indent<'line>(line: &'line str, indent: &str) -> &'line str {
     &line[indent.len()..]
 }
 
+#[cfg(target_os = "macos")]
+const DYNAMIC_LIBRARY_EXTENSION: &str = ".dylib";
+#[cfg(all(unix, not(target_os = "macos")))]
+const DYNAMIC_LIBRARY_EXTENSION: &str = ".so";
+#[cfg(windows)]
+const DYNAMIC_LIBRARY_EXTENSION: &str = ".dll";
```

Comment:
You could make this slightly less repetitive by using the `cfg!` macro (as long as if-expressions can be `const`, don't know if that's true): https://doc.rust-lang.org/std/macro.cfg.html

Reply:
Ah interesting ok updated to use `cfg!()` for all of these

> as long as if-expressions can be `const`, don't know if that's true

oh ok looking at those docs that actually isn't required because it doesn't `const`-eval the whole `if`-block, just replaces the `cfg!(...)` -> boolean